Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/test-run-details/TestRunDetails.tsx

```diff
@@ -52,7 +53,29 @@ const TestRunDetails = ({ testRun }: ITestRunDetailsProps) => {
         return testRunText;
     }, [ testRun ]);
 
-    const renderTimeAndMemoryUsed = useCallback(() => (
+    const renderLabel = useCallback(
```

Comment:
I think renderResultTypeLabel  naming will indicate better what the label is for

Reply:
Renamed.